recipes/numpy_indexed/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: numpy-indexed
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/EelcoHoogendoorn/Numpy_arraysetops_EP.git
+  git_tag: {{ version }}
+  
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+  
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pyyaml
+    - future
+  run:
+    - python
+    - numpy
+    - future
+
+test:
+  commands:
+    - py.test --pyargs numpy_indexed
+  imports:
+    - numpy_indexed
+  requires:
+    - pytest
+  
+about:
+  home: https://github.com/EelcoHoogendoorn/Numpy_arraysetops_EP
+  license: Freely Distributable
```

Comment:
I cant say i am well versed on the subject; ive discussed this with ivo, and we are thinking fof going for LGPL3; do you have any thoughts on the matter?